Removes compilation flag that does not apply to ARM CPUs

Author: Ess Ridley (JSC)

--- a/mf/Makefile.defs.linux-gomp
+++ b/mf/Makefile.defs.linux-gomp
@@ -11,7 +11,7 @@ PREFIX           = $(TOPDIR)/../../install/sionlib_linux_gomp
 # Platform Settings
 #------------------------------------------------------------------------------
 PREC   = 64
-PFLAG  = -m$(PREC)
+PFLAG  =
 AFLAG  =

 PLAT     = linux
diff --git a/mf/Makefile.defs.linux-gomp10 b/mf/Makefile.defs.linux-gomp10
index 5275d8b8..414ad09a 100644
--- a/mf/Makefile.defs.linux-gomp10
+++ b/mf/Makefile.defs.linux-gomp10
@@ -11,7 +11,7 @@ PREFIX           = $(TOPDIR)/../../install/sionlib_linux_gomp
 # Platform Settings
 #------------------------------------------------------------------------------
 PREC   = 64
-PFLAG  = -m$(PREC)
+PFLAG  =
 AFLAG  =

 PLAT     = linux
